.circleci/config.yml

```diff
@@ -1,27 +1,31 @@
 version: 2.1
 
+
 orbs:
-  python: circleci/python@0.2.1
+  node: circleci/node@2.0.2
   coveralls: coveralls/coveralls@1.0.6
 
 jobs:
-  build-and-test:
-    docker:
-      - image: circleci/python:3.8-node
+  test:
+    parameters:
+      node-version:
+        type: string
+    executor: node/default
+    working_directory: ~/slic-watch/serverless-plugin
     steps:
-      - checkout
-      - setup_remote_docker:
-          version: 19.03.13
-      - run:
-          name: Install Dependencies
-          command: make deps
-      - run:
-          name: Test
-          command: make test lint
+      - checkout:
+          path: ~/slic-watch
+      - node/install:
+          node-version: << parameters.node-version >>
+      - node/install-packages:
+          app-dir: ~/slic-watch/serverless-plugin
+      - run: npm run cov
       - coveralls/upload:
-          path_to_lcov: ./.coverage
-
+          flag_name: << parameters.node-version >>
 workflows:
-  main:
+  workflow:
```

Comment:
Should we add a job for the npm publish here too?

Reply:
Agreed. I'll create an issue to do this separately.